airflow/providers/google/cloud/example_dags/example_dataproc.py

```diff
@@ -23,13 +23,14 @@
 import os
 
 from airflow import models
+from airflow.contrib.operators.dataproc_operator import DataprocClusterCreateOperator
```

Comment:
I dont think this is something we want to do.

Our documntation and examples should always support latest airflow version not EOL versions.
If users have trouble with 1.10 they can ask in Disscusion and get support.
It should not be part of the code base.

Reply:
Information about the upgrading to the latest version should be to UPDATING.md. https://github.com/apache/airflow/blob/main/UPDATING.md#changes-in-google-provider-package

Reply:
@eladkal  @mik-laj 
Agree, I will information about older version from here and move it to updating.md
Also this is not only useful while upgrading, In general its much easier to generate CLUSTER_CONFIG from **ClusterGenerator** API